### PR TITLE
Improved config.yml

### PR DESCRIPTION
**Pull request by Chris6ix**

### CODE REVIEW

**Review by Ghost-chu — APPROVED** [review]

File: src/main/resources/config.yml [hunk 1]
```diff
@@ -61,45 +61,47 @@ config-version: 129
...
-  #Should we log balance before and after in shop trade to Plugins\QuickShop\qs.log?
+  #Should QuickShop log balance before and after when trading?
   log-balance: true
-  #The target size to split large files (MB) to Plugins\QuickShop\logs folder
+  #The target size (in MB) to split log files to the "plugins\QuickShop\logs" folder.
   file-size: 10.0
-#Some eco plugins will caching player balance, sometime will cause insufficient balance
-#This workaround using getBalance to refresh it, but is time-costing
-#So turn on this ONLY when you have this problem
+#The log file is located at "plugins\QuickShop\qs.log".
```
Comment (sandtechnology) [inline src/main/resources/config.yml:101]: It should be added at line 93

File: src/main/resources/config.yml [hunk 2]
```diff
@@ -108,240 +110,263 @@ whole-number-prices-only: false
...
-#DO NOT TOUCH THIS IF YOU DIDN'T KNOW WHAT WILL HAPPEN. ASK FOR SUPPORT BEFORE TOUCH THIS!
+#DO NOT TOUCH THIS IF YOU DON'T KNOW WHAT IT DOES. ASK FOR SUPPORT BEFORE TOUCHING THIS!
+#DO NOT TOUCH THIS IF YOU DON'T KNOW WHAT IT DOES. ASK FOR SUPPORT BEFORE TOUCHING THIS!
+#DO NOT TOUCH THIS IF YOU DON'T KNOW WHAT IT DOES. ASK FOR SUPPORT BEFORE TOUCHING THIS!
 economy-type: 0
 
-#The decimal format the double output, includes player money output
-#Default is #,###.##
-# whether to use decimal format output.
+#Whether to use decimal format to dislplay money.
```
Comment (sandtechnology) [inline src/main/resources/config.yml:139]: dislplay-> display

File: src/main/resources/config.yml [hunk 3]
```diff
@@ -108,240 +110,263 @@ whole-number-prices-only: false
...
   fee-for-price-change: 50
 
-  #Should we try and lock their shops from other players, so people can't steal from them?
+  #Should QS try to lock shops for other players?
+  #This prevents players from stealing from them.
   lock: true
 
-  #Should we disable the quick way of creating shop? (click a chest with item)
+  #Should QS disable the quick creation of shop?
+  #To quickly create a shop click a chest with an Item.
```
Comment (sandtechnology) [inline src/main/resources/config.yml:242]: To quickly create a shop by clicking a chest with an Item?

File: src/main/resources/config.yml [hunk 4]
```diff
@@ -108,240 +110,263 @@ whole-number-prices-only: false
...
   pay-unlimited-shop-owners: false
 
-  #Should we place display items on the chests? This may cause duped items, especially for modded servers.
+  #Should we place display items on the chests? 
+  #This may cause duped items, especially for modded servers!
   display-items: true
 
-  #The plugin will check if the display item is in a valid position every specified amount of ticks. 0 to disable.
+  #QuickShop will check if the display item is in a valid position every specified amount of ticks. 
+  #Set to 0 to disable.
```
Comment (sandtechnology) [inline src/main/resources/config.yml:269]: Set to 0 to disable it?

File: src/main/resources/config.yml [hunk 5]
```diff
@@ -108,240 +110,263 @@ whole-number-prices-only: false
...
   ignore-unlimited-shop-messages: false
 
-  #Allow auto fetching player shops, if you disable this, players need to use /qs fetchmessage to fetch the shops history messages. (Disabling it may cause database issues.)
+  #Allow auto fetching of player shops.
+  #If you disable this, then the players need to use "/qs fetchmessage" to fetch the shops history messages. 
+  #Disabling it may cause database issues!
   auto-fetch-shop-messages: true
 
-  #Ignore cancel chat event, can improve compatibility with other chat plugin in some cases.
+  #Ignore cancel chat event, can sometimes improve compatibility with some chat plugins.
```
Comment (sandtechnology) [inline src/main/resources/config.yml:337]: cancel->cancelled

**Review by sandtechnology — CHANGES_REQUESTED** [review]

File: src/main/resources/config.yml (hunk 4 above)
Comment (Chris6ix) [reply src/main/resources/config.yml:269]: "0 to disable." -> "Set to 0 to disable."

File: src/main/resources/config.yml (hunk 3 above)
Comment (Chris6ix) [reply src/main/resources/config.yml:242]: "(click a chest with item)" -> "To quickly create a shop click a chest with an Item."

Improved explanation of what "the quick way of creating shop" is...

File: src/main/resources/config.yml (hunk 2 above)
Comment (Chris6ix) [reply src/main/resources/config.yml:139]: sry

File: src/main/resources/config.yml (hunk 5 above)
Comment (Chris6ix) [reply src/main/resources/config.yml:337]: ohh i thought "cancel chat event" was what its name is...

File: src/main/resources/config.yml [hunk 6]
```diff
@@ -353,313 +378,330 @@ shop:
...
+    #Which checks does a player need to pass to create a shop?
+    #You can use multiple checks at the same time.
+    #Priority: UP > DOWN
     #
-    #Avaliable field:
-    #BUILD = Player must have ability to build at the target claim.
-    #CONTAINER_ACCESS = Player must can access container.
-    #ACCESS = Player must have ability to access the target claim.
+    #Available fields:
+    #BUILD = The player must have the ability to build ain the target claim.
```
Comment (sandtechnology) [inline src/main/resources/config.yml:657]: ain -> at

File: src/main/resources/config.yml [hunk 7]
```diff
@@ -353,313 +378,330 @@ shop:
...
+    #Which checks does a player need to pass to trade with a shop?
+    #You can use multiple checks at the same time.
+    #Priority: UP > DOWN
     #
-    #Avaliable field:
-    #BUILD = Player must have ability to build at the target claim.
-    #CONTAINER_ACCESS = Player must can access container.
-    #ACCESS = Player must have ability to access the target claim.
+    #Available fields:
+    #BUILD = The player must have the ability to build ain the target claim.
```
Comment (sandtechnology) [inline src/main/resources/config.yml:667]: ain -> at

File: src/main/resources/config.yml [hunk 8]
```diff
@@ -685,23 +727,24 @@ integration:
...
+#The protection that a shop should check.
 protect:
   explode: true
   hopper: true
   entity: true
-#This option allow you custom item max stacksize.
-# (* means all item, if not match all stacksize options then will use it)
-# Example:
+#This option allows you to set custom item max stacksizes.
+#If nothing matches, then all stacksize options will use *.
```
Comment (sandtechnology) [inline src/main/resources/config.yml:745]: If nothing matches, then will use all items option (*).

**Review by sandtechnology — CHANGES_REQUESTED** [review]

**Comment by sandtechnology** [comment]
Thanks for contributing!

**Pull request merged**